django-tethne/models.py

```diff
@@ -0,0 +1,163 @@
+from django.db import models
+
+
+
+"""
+Model Name = Source
+
+Source represents the source of the Corpus
+Example 1. JSTOR, 2. ZOTERO or 3. WOS
+"""
+class Source(models.Model):
+    source_value = models.CharField(max_length=35, null=True)
+
+"""
+
+Model name = Corpus
+Will store the corpus details, for example source, the number of papers and label.
+"""
+
+class Corpus(models.Model):
+    source = models.ForeignKey(Source)
```

Comment:
Let's add the ``related_name`` parameter on ForeignKey relations: https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.ForeignKey.related_name

e.g.

```python
source = models.ForeignKey(Source, related_name='corpora')
```